test/demo/compute-resource.yml

```diff
@@ -0,0 +1,20 @@
+---
+- hosts: localhost
+  tasks:
+    - name: vmware compute resource
+      foreman_compute_resource:
+        name: example_compute_resource
+        datacenter: ax01
+        locations:
+        - Munich
+        organizations:
+        - ATIX
+        provider: vmware
+        provider_auth:
+          url: vsphere.example.com
+          user: admin
+          password: secret
+        server_url: vsphere.example.com
```

Comment:
Perhaps this should be `https://foreman.example.com` to match all the other demo playbooks.